WcaOnRails/app/models/competition.rb

```diff
@@ -1,5 +1,9 @@
 class Competition < ActiveRecord::Base
   self.table_name = "Competitions"
+  # FIXME Tests fail with "Unknown primary key for table Competitions in model Competition."
+  #       when not setting the primary key explicitly. I have
+  #       no clue why... (th, 2015-09-19)
+  self.primary_key = "id"
```

Comment:
That's interesting. I would not have expected your changes to affect our runtime at all.

Could you add a comment to https://github.com/cubing/worldcubeassociation.org/issues/91 so we remember to get to this?